src/main/java/org/carlmontrobotics/commands/AlignToApriltag.java

```diff
@@ -21,18 +22,25 @@ public class AlignToApriltag extends Command {
     public final Drivetrain drivetrain;
     private Limelight limelight;
 
-    public final PIDController rotationPID = new PIDController(thetaPIDController[0], thetaPIDController[1],
-            thetaPIDController[2]);
+    public final PIDController rotationPID = new PIDController(
+                    SmartDashboard.getNumber("apriltag align kp",
+                                    thetaPIDController[0]),
+                    SmartDashboard.getNumber("apriltag align ki",
+                                    thetaPIDController[1]),
+                    SmartDashboard.getNumber("apriltag align kd",
+                                    thetaPIDController[2]));
```

Comment:
This is also only called once when the robot is starting up, we want to be able to tune the values while the robot is running

Reply:
i put it in periodic in limelight.java

Reply:
if you use `put` in periodic, it will keep overriding the value making it impossible to actually edit the value. Also merely changing the value in SmartDashboard doesn't actually change the pid constants. You need to call pidController.setP([insert value]) in periodic for it to actually change.